boards/arm/stm32f746g_disco/board.cmake

```diff
@@ -1 +1,8 @@
-include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+set_ifndef(STLINK_FW stlink)
```

Comment:
There's not really a need for these sorts of variables anymore, is there? You can `west flash --runner jlink` or `west flash --runner openocd`. The default runner can still be set to jlink.